src/js/components/tour/plan.tsx

```diff
@@ -0,0 +1,126 @@
+} from 'react-joyride';
+
+import PopoverHeader from '~img/popover-header.png';
+
+interface Props {
+  run: boolean;
+  onClose: () => void;
+}
+
+const PlanTour = ({ run, onClose }: Props) => {
+  const [helpers, setHelpers] = useState<StoreHelpers | null>(null);
+
+  const steps: Step[] = [
+    {
+      target: '.tour-create-epic',
+      title: i18n.t('Create Epics to group Tasks'),
+      content: (
+        <Trans i18nKey="tourCreateEpic">
+          Epics are groups of related Tasks, representing larger changes to the
+          Project. You can invite multiple Collaborators to your Epic and assign
+          different people as Developers and Testers for each Task.
+        </Trans>
+      ),
+      placement: 'right',
+      disableBeacon: true,
+    },
+    {
+      target: '.tour-walkthroughs',
+      title: i18n.t('Review walkthroughs any time'),
+      content: (
+        <Trans i18nKey="tourWalkthroughs">
+          If you would like to review this or any other walkthrough again,
+          simply click the question mark. There is also a self-guided tour with
+          more object and action explanations.
+        </Trans>
+      ),
+      placement: 'right',
+      disableBeacon: true,
+    },
+  ];
+
+  const handleCallback = useCallback(
+    (data: CallBackProps) => {
+      const { action, status } = data;
+      const finished: StatusType[keyof StatusType][] = [
+        STATUS.FINISHED,
+        STATUS.SKIPPED,
+      ];
+      if (finished.includes(status) || action === ACTIONS.CLOSE) {
+        /* istanbul ignore else */
+        if (helpers) {
+          helpers.close();
+        }
+        onClose();
+      }
+    },
+    [onClose, helpers],
+  );
+
+  return (
+    <Joyride
+      steps={steps}
+      run={run}
+      locale={{
+        back: i18n.t('Back'),
+        close: i18n.t('Close'),
+        last: i18n.t('Close'),
+        next: i18n.t('Next'),
+      }}
+      continuous
+      showProgress
+      disableScrolling
+      getHelpers={setHelpers}
+      callback={handleCallback}
+      styles={{
+        options: {
+          arrowColor: '#032e61',
+          backgroundColor: '#032e61',
+          textColor: '#ffffff',
+          primaryColor: '#0070d2',
+        },
+        tooltip: {
+          padding: '0',
+        },
+        tooltipContainer: {
+          textAlign: 'left',
+        },
+        tooltipContent: {
+          fontSize: '.8125rem',
+          padding: '1rem',
+        },
+        tooltipTitle: {
+          backgroundColor: '#164a85',
+          backgroundImage: `url(${PopoverHeader})`,
+          backgroundRepeat: 'no-repeat',
+          backgroundPosition: 'bottom',
+          backgroundSize: 'contain',
+          borderBottom: '1px solid #ffffff',
+          fontSize: '1.25rem',
+          fontWeight: 300,
+          padding: '0.75rem 1rem',
+          textShadow: '0 0 4px #032e61',
+        },
+        tooltipFooter: {
+          margin: 0,
+          padding: '1rem',
+        },
+      }}
```

Comment:
I'm making some changes I haven't pushed yet to the buttons, but I wonder if you could help @jgerigmeyer? Is there a way to edit the buttonBase instead of each of the 4 button types? 
https://github.com/gilbarbara/react-joyride/blob/c33746d542fd8203d2e85ed07a0c07753c3cc86f/src/styles.js#L15

When I add buttonBase inside the styles={{ }}  here it turns a different color and underlines everything in red. I assume I can't add this base style within this styles block... but how is it added then?

Also, These styles will be used on all of the tours, so should they be moved into a different file so they arent duplicated in the play and help walk throughs as well?

Or, is there a way to add a lighting design class to each of the elements so we don't have to override the styles? Or is it too tricky to access the markup like that? I don't mind which way we go, but I assume this style duplication will be a bummer to maintain if there are changes down the road.

Reply:
> I'm making some changes I haven't pushed yet to the buttons, but I wonder if you could help @jgerigmeyer? Is there a way to edit the buttonBase instead of each of the 4 button types?
> https://github.com/gilbarbara/react-joyride/blob/c33746d542fd8203d2e85ed07a0c07753c3cc86f/src/styles.js#L15
> 
> When I add buttonBase inside the styles={{ }} here it turns a different color and underlines everything in red. I assume I can't add this base style within this styles block... but how is it added then?

@stacyk 

No, we can't directly override the `buttonBase` to style all buttons at once. That's only defined internally in react-joyride, and then the default Joyride styles override those individually, and then our styles override those individually as well. So we'd have to style each of them separately.

> Also, These styles will be used on all of the tours, so should they be moved into a different file so they arent duplicated in the play and help walk throughs as well?

Sure, I think you're 💯% right... but this PR is just for the first walkthrough, so I'd say let's worry about abstracting patterns once we have patterns to abstract. 😄 

> Or, is there a way to add a lighting design class to each of the elements so we don't have to override the styles? Or is it too tricky to access the markup like that?

I don't think so, no. I mean, we can choose to override _all_ the markup if we want -- but I'd rather not go that route unless we really want to. We can't just attach classes, without rolling our own markup entirely. And this seems pretty close to me...

> I don't mind which way we go, but I assume this style duplication will be a bummer to maintain if there are changes down the road.

Yeah, I get that. I'd lean toward creating our own style definition here (using their markup), then when we add the next walkthrough we'll move this to its own file and use it for all walkthroughs. Does that work?